autotest/pst_from_tests.py

```diff
@@ -305,7 +304,8 @@ def freyberg_prior_build_test():
                  longnames=True, spatial_reference=sr,
                  zero_based=False)
     pf.extra_py_imports.append('flopy')
-    pf.mod_sys_cmds.append("which python")
+    if "linux" in platform.platform().lower():
+        pf.mod_sys_cmds.append("which python")
```

Comment:
Think I just added this when I was having some pathing issues can probs be dropped if not useful.